Concentration/Concentration.swift

```diff
@@ -6,24 +6,39 @@
 //
 
 import Foundation
+import UIKit
 
 final class Concentration {
 
     typealias CardIndex = Array<Card>.Index
 
     // MARK: - Properties -
 
-    var cards: [Card] = []
+    private (set) var cards: [Card] = []
+    private (set) var flipCount: Int = 0
+    private (set) var score: Int = 0
+
+    let cardBackgroundColor: UIColor
+
+    private var emojiChoices: [String]
+    private var emoji = [Int: String]()
```

Comment:
for me this way of setting/getting data is questionable. However I didn't want to refactor initial lecturer logic so decided to use the same way of setting/getting emojis